cargo-sqlx/.gitignore

```diff
@@ -1,4 +1,2 @@
 /target
-/migrations
 Cargo.lock
-.env
```

Comment:
Sorry, `.env` we do want to keep. It was just that `.env-pg` line and `*/*.sql` that didn't really belong.

Reply:
No problem, should be fixed now :)